src/utils/string-utils.js

```diff
@@ -1,19 +1,18 @@
-export function getSearchTermIndex(text, searchTerm) {
-  return text.toLowerCase().indexOf(searchTerm.toLowerCase());
-}
-
-export function splitAt(text, ...indices) {
-  const result = [];
-  let lastIndex = 0;
-  for (const index of indices) {
-    result.push(text.substr(lastIndex, index - lastIndex));
-    lastIndex = index;
-  }
-  result.push(text.substr(lastIndex));
-  return result;
-}
+import escapeStringRegexp from 'escape-string-regexp';
 
-export default {
-  getSearchTermIndex,
-  splitAt
+const HTML_REPLACEMENT_MAP = {
+  '&': '&amp;',
+  '<': '&lt;',
+  '>': '&gt;',
+  '"': '&quot;'
 };
+
+const HTML_ESCAPE_TEST_PATTERN = new RegExp(`[${Object.keys(HTML_REPLACEMENT_MAP).map(escapeStringRegexp).join('')}]`);
+
+const HTML_ESCAPE_REPLACEMENT_PATTERN = new RegExp(HTML_ESCAPE_TEST_PATTERN.source, 'g');
+
+export function escapeHtml(str) {
```

Comment:
Maybe we should then use this method also in the `page-renderer` and `mail-service` and remove the external `htmlescape` dependency 🤔 What do you think?

Reply:
They don't do exactly the same though. https://www.npmjs.com/package/htmlescape escapes JSON for usage as an object literal inside of a `<script>` tag. They escape slightly different characters.